src/chrome/content/rules/administrator.de.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="administrator.de">
+<target host="*.administrator.de" />
+<rule from="^http:" to="https:" />
+<test url="http://www.administrator.de/" />
```

Comment:
Suggest to add:
```
<rule from="^http://administrator\.de/"
	to="https://www.administrator.de/" />
```
To illustrate, without above lines, the redirection will be:
`http://administrator.de -> http://www.administrator.de -> https://www.administrator.de`
with above rules:
`http://administrator.de -> https://www.administrator.de`
In the end, you still get https://www.administrator.de though, so it's not strictly essential.
If you decide to take my suggestion, keep line 4, as the above lines require it.